public/js/application.js

```diff
@@ -7,27 +7,50 @@ $(document).ready(function() {
       method: 'get'
     })
     .done(function(response) {
-      console.log(response)
       $('#TODO_area').append(response)
     })
-
-    $(this).remove();
+    $(this).hide();
     // $('#TODO_area').append("");
   })
-  // $('#TODO_area').on('submit', 'form[id="create_new_todo"]', function(e){
-  //   console.log('working');
-  //   e.preventDefault();
-  //   var info = $(this).serialize();
-  //   $.ajax({
-  //     method: "POST",
-  //     url: "/todos",
-  //     data: info
-  //   })
```

Comment:
$(document).ready(function() {
   bindEvents();
}

var bindEvents = function() {
   $('#TODO_area').on('submit', 'form[id="create_new_todo"]',createNewTodo)
}

var createNewTodo =  function(e){
 +    e.preventDefault();
 +    var info = $(this).serialize();
 +    $.ajax({
 +      method: "POST",
 +      url: "/todos",
 +      data: info
 +    })
 +    .done(function(new_todo){
 +      $('#create_new_todo').remove();
 +      $('#new_todo').show();
 +      $('#TODO_area').append(new_todo);
 +    })
 +    .fail(function(){
 +      console.log("bad");
 +    })
 +  })